compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/CMakeLists.txt

```diff
@@ -26,6 +26,10 @@ iree_cc_library(
     iree::target::amd-aie::Transforms
 )
 
+add_dependencies(iree_target_amd-aie_Target_AIETargets
+        iree-amd-aie_aie_runtime_amdaie_xclbinutil
+        iree-amd-aie_aie_runtime_bootgen)
+
```

Comment:
One thing that I'm not handling correctly here is making these utilities available in the build directory (i.e., in `build/tools`). I tried to get it to work using IREE's CMake functions but couldn't get it. Can someone give me a tip on how this is done.